api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -296,6 +296,8 @@ private RdrWorkspace toRdrWorkspace(DbWorkspace dbWorkspace) {
     rdrWorkspace.setFindingsFromStudy(dbWorkspace.getAnticipatedFindings());
     rdrWorkspace.setReviewRequested(dbWorkspace.getReviewRequested());
 
+    rdrWorkspace.setFocusOnUnderrepresentedPopulations(
+        dbWorkspace.getSpecificPopulationsEnum().size() > 0);
```

Comment:
Do we have a guarantee that the getSpecificPopulationsEnum value will be non-null? Looking at the existing code, we seem to be assuming it's non-null. Just wondering where that expectation comes from.

Reply:
This will non-null since populationDetails  is initialized to a HashSet  [DbWorkspace](https://github.com/all-of-us/workbench/blob/cea86f34bea7eb37a0948ad5a7c1aa948c6035f6/api/src/main/java/org/pmiops/workbench/db/model/DbWorkspace.java) L 102

If this is ever change to null, RdrExportServiceImplTest.exportWorkspace_successful will fail with null pointer exception